UnityProject/Assets/Scripts/UI/Systems/UIManager.cs

```diff
@@ -625,4 +626,15 @@ private IEnumerator WaitForStrandedVideoEnd()
 
 		ChatUI.Instance.OpenChatWindow();
 	}
-}
+
+	public static void LogPlayerAction(string incidentTime, PlayerAlertTypes alertType, PlayerInfo perp, string message)
```

Comment:
feels odd to have this in this component

Reply:
Imo no. And if we were in an ECS environment this would be a common thing to have.

Reply:
that can't be more wrong. in ECS you are even more cautelous with where you put stuff. Logging player actions has nothing to do with the UIManager, it isn't a responsibility of the UIManager to pay attention to logs and players actions.